src/org/ods/component/Pipeline.groovy

```diff
@@ -278,33 +280,14 @@ class Pipeline implements Serializable {
     if (!this.bitbucketNotificationEnabled) {
       return
     }
-    if (!context.jobName || !context.tagversion || !context.credentialsId || !context.buildUrl || !context.bitbucketUrl || !context.gitCommit) {
-      logger.info "Cannot set BitBucket build status to ${state} because required data is missing!"
+    if (!context.jobName || !context.tagversion || !context.buildUrl || !context.gitCommit) {
```

Comment:
@michaelsauter  - couldnt we move this into the bitbucket service (setBuildStatus) .. this is a non reusable check .. best case .. we only have one line in the pipeline - (bitbucketNo..?)

Reply:
I left this here on purpose. The service deals with Bitbucket and has no knowledge of the `context`. Of course we can parse every bit into it, but that does not seem right to me. Also, the service has no knowledge of what a build name should be - it is assembled in the pipeline, and therefore the pipeline needs `jobName` and `tagversion` which the service does not need anymore.